app/src/components/SearchBar.jsx

```diff
@@ -2,17 +2,23 @@ import React, { useState } from "react";
 import "../stylesheets/searchBar.scss";
 import closeSvg from '../assets/close.svg'
 import SearchSvg from '../assets/search.svg'
+import useFetch from "../hooks/useFetch";
+import { Link } from 'react-router-dom';
 
 
-function SearchBar({ placeholder, data }) {
+
+function SearchBar({ placeholder }) {
     const [filteredData, setFilteredData] = useState([]);
     const [wordEntered, setWordEntered] = useState("");
 
+    const { error, isPending, data: artists } = useFetch("/data/artists");
+
+
     const handleFilter = (event) => {
         const searchWord = event.target.value;
         setWordEntered(searchWord);
-        const newFilter = data.filter((value) => {
-            return value.band.toLowerCase().includes(searchWord.toLowerCase());
+        const newFilter = artists.filter((value) => {
```

Comment:
what is value here? an artist?

Reply:
I nuläget är value den artisten som hittats genom sökresultaten. Detta ska även vara venues och events. Därav value, då det är värdet som matchar sökningen. Kan justeras vid bättre ordval

Reply:
Vi byter till artist i nuläget. Lägger upp en ticket på att koppla in events och venues till sökfunktionen.